### ISSUE TITLE
"Stale" issues being closed?

### ISSUE THREAD

**Issue by janssen** [kivy/kivy#4783]
What makes an issue "stale"?  Typically in my experience issues are never closed unless they are resolved one way or another.  That's why the issue tracker exists, isn't it?  To keep track of things that might otherwise fall by the wayside and be forgotten?  Is there a problem with too many issues?

**Comment by dessant** [comment]
I agree.

On the other hand the bug tracker becomes useless when we are drowned in issues that nobody has intention to act upon in the foreseeable future, nor are they critical. For *some* feature requests, submitting pull requests could become the preferred method, and even then the current team may not be able to keep up with maintenance. We're simply not enough, nor do we seem to have enough time.

The closures you have seen yesterday can be considered a reality check.

**Comment by FluxIX** [comment]
Submitting pull requests would be useful _if_ they were processed in a reasonable amount of time, but it seems the maintenance team is having difficultly doing so currently. I disagree with closing bugs because of an inability to investigate or fix the bugs.

**Comment by dessant** [comment]
We have an easier time reviewing pull requests, than starting from working on implementing new features.
If you feel some of the recently closed issues are both important and actionable, please create new PRs for them, and we will do our best and try to process them.

https://github.com/kivy/kivy/search?p=1&q=stale&type=Issues&utf8=%E2%9C%93

**Comment by dessant** [comment]
I'd also like to invite everyone interested in contributing to take a look at the current list of pull requests, simply running them and reporting back if it they work would mean a lot.

https://github.com/kivy/kivy/pulls?q=is%3Aopen+is%3Apr

**Comment by matham** [comment]
> I disagree with closing bugs because of an inability to investigate or fix the bugs.

I do agree that we should not close bugs if we cannot investigate or fix them. But I don't think the issues we closed really fall into that category. Most of them were old and are issues that were never reproduced or seem likely to be user error that is hard to figure out on github given their age. Others are feature requests that are unlikely to implemented unless there's a PR. So there's no reason to have an issue open for them.

And of course if the issue shouldn't have been closed it can always be reopened.
